pkg/controllers/clusterresourceplacement/placement_status_test.go

```diff
@@ -6235,48 +7245,51 @@ func TestBuildClusterResourceBindings(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			crp := fleetv1beta1.ClusterResourcePlacement{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: testCRPName,
-				},
-			}
 			policySnapshot := fleetv1beta1.ClusterSchedulingPolicySnapshot{
```

Comment:
use SchedulingPolicySnapshot for the RP tests?

Reply:
that's true, but better to add a comment to provide context otherwise, it seems confusing when reading the code.